sdks/java/io/common/src/main/java/org/apache/beam/sdk/io/common/synthetic/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** Transforms for performing Synthetic Operations in Apache Beam pipelines. */
+package org.apache.beam.sdk.io.common.synthetic;
```

Comment:
I am partially 'partagé' on having this in common. If the goal is to have a synthetic IO maybe the need is specific so better be an independent IO, no ? And then we can migrate some features from other generators there, and write just ParDo transformations on this e.g. to generate data for Nexmark or examples.

Reply:
I think that makes a lot of sense. I added a new component. WDYT?